pages/about.html

```diff
@@ -1,12 +1,73 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Sana</title>
+    <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+    <header>  
+        <h1>A Little About ME!</h1>
+        <div class="top">
+            <a href="index.html"> Swiftie Website</a>
+            <a href="portfolio.html"> My Portfolio</a>
+        </div>
+    </header>
+
+<section>
+    <div class="intro">
+        
+        <img alt="A picture of me on a kayak laughing" src="../assets/my_photo.png"/>
+        
+        <div>
```

Comment:
Using a section tag here makes sense. You can put the class "intro" on the section tag. We should try to avoid using non semantic elements like div unless we absolutely need it. When you go to write a div or span tag, ask yourself what it achieves or represents. See if you can replace it with a more semantic element.